content/changelog/2024-10-04.md

```diff
@@ -0,0 +1,63 @@
+---
+description: tbd
+---
+
+### Sign up for Neon Deploy
+
+Join us online on **October 30th** at **10:00 AM** PT to learn how Neon empowers developers to ship faster with Postgres. Tune in for Product updates and technical deep dives. [Sign up](https://neon.tech/deploy) today!
+
+![join us at Neon Deploy](/docs/relnotes/neon-deploy.png)
+
+### Neon on Azure is open for beta
+
+Neon on Azure is now available for public beta! Create a [new project](https://console.neon.tech/app/projects/new?provider=azure) now and try it out.
+
+![azure project creation globe](/docs/relnotes/azure_beta.png)
+
+### More projects
+
+Customers with [database-per-tenant](https://neon.tech/use-cases/database-per-tenant), developer platforms, and AI Agents told us they needed more projects: we've expanded the project limits across all paid Neon plans.
+
+New limits:
+
+- **Launch Plan**: Up to 100 projects
+- **Scale Plan**: Up to 1,000 projects
+- **Business Plan**: Up to 5,000 projects
+
+A reminder, last week we introduced the experimental [@neondatabase/toolkit](https://github.com/neondatabase/toolkit), which allows you to spin up a Postgres database in seconds and run SQL queries &#8212; ideal for quick setups and testing environments, and for taking advantage of your new, higher project limits.
+
+### **ISO27110 & ISO27701 compliance**
+
+These new certifications add to our growing list of compliance achievements. For more about Neon's compliance milestones, see [Compliance](/docs/security/compliance).
+
+### Database limits per branch
+
+In case you missed last week's heads up, but this week we've started enforcing a limit of 500 roles and 500 databases per branch. If you have any concerns about these new limits, please reach out to the Neon Support team. Previously, these limits were recommended but not enforced.
```

Comment:
```suggestion
In case you missed last week's heads up &#8212; this week we've started enforcing a limit of 500 roles and 500 databases per branch. If you have any concerns about these new limits, please reach out to the Neon Support team. Previously, these limits were recommended but not enforced.
```